map/mode.h

```diff
@@ -0,0 +1,98 @@
+#ifndef __MAP_MODE_H__
+#define __MAP_MODE_H__
+
+#include "mam/api/api.h"
+
+#define MSS_DEPTH 1
```

Comment:
Instead of hardcode, can we explain the consideration in advance?

Reply:
I would like to make mss an argument and even able to auto set the size base on length of message/payload. And since it's going to be under `map`, I think it would be easier to roll back to macro first then implement the actual module in another PR.